github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java

```diff
@@ -82,6 +87,42 @@ public boolean apply(GitHubPRCause cause) {
                 }
             }
 
+            if (trigger.isAbortRunning()) {
+                Computer[] computers = getJenkinsInstance().getComputers();
+                for (Computer computer : computers) {
+                    if (isNull(computer)) {
+                        continue;
+                    }
+
+                    List<Executor> executors = computer.getExecutors();
+
+                    for (Executor executor : executors) {
+                        if (isNull(executor)) {
+                            continue;
+                        }
+
+                        if (executor.isBusy()) {
+                            Queue.Executable executable = executor.getCurrentExecutable();
```

Comment:
Can check if this `instanceof Run`, if you are going to be doing `Run`-specific logic anyway. Then you do not need to deal with `Queue.Task`.